tests/ui_tools/test_boxes.py

```diff
@@ -179,11 +184,15 @@ def test_keypress_ENTER(self, panel_search_box,
         size = (20,)
         panel_search_box.panel_view.view.controller.editor_mode = True
         panel_search_box.panel_view.log = log
+        panel_search_box.set_caption("")
         panel_search_box.edit_text = "key words"
 
         panel_search_box.keypress(size, enter_key)
 
         # Update this display
+        # We can't test for the styled version?
+        # expected_caption = [('filter_results', 'Search Results'), ' ']
```

Comment:
Just to clarify, did you miss this (or) is this on purpose?

Reply:
I left this in as a note - perhaps I should change to FIXME - but couldn't find an obvious way to test for the applied style of the caption.

Reply:
Adjusted slightly to indicate this.